src/api/VoteMonitor.Api.Core/Handlers/IPaginatedQueryHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace VoteMonitor.Api.Core.Handlers
+{
+    public interface IPaginatedQueryHandler
```

Comment:
i don't think we need this interface at all

Reply:
Hi @idormenco , thanks for your feedback: I've removed it